app/src/main/java/com/github/se/travelpouch/ui/overview/Overview.kt

```diff
@@ -0,0 +1,126 @@
+package com.github.se.travelpouch.ui.overview
+
+// import com.github.se.travelpouch.ui.navigation.NavigationActions
+import android.annotation.SuppressLint
+import android.icu.text.SimpleDateFormat
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.KeyboardArrowRight
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.Card
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.github.se.travelpouch.model.TravelContainer
+import com.github.se.travelpouch.ui.navigation.NavigationActions
+import java.util.Locale
+
+/**
+ * Composable function for the Overview screen.
+ *
+ * @param navigationActions Actions for navigation.
+ * @param travelContainers List of travel containers to display.
+ */
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter") // TODO useful?
+@Composable
+fun OverviewScreen(navigationActions: NavigationActions, travelContainers: List<TravelContainer>) {
```

Comment:
I think that you cannot use the collectAsState() funtion for now, to work on the list of travelContainners instead of passing as a parameter, because there are no repository or model view to communicate with firebase. But later it might be better to do so.

Reply:
Hey, well now you can!
To elaborate : it is also necessary because it's your view that will pass the viewmodel to the underlying AddTravelScreen using composition